src/components/Global/Input/index.tsx

```diff
@@ -28,8 +32,12 @@ const Input = ({
         required={required}
         placeholder={placeholder}
         maxLength={maxLength}
+        minLength={minLength}
         value={input}
-        onChange={event => setInput(event.target.value)}
+        onChange={event => {
+          setInput(event.target.value);
```

Comment:
If you're using the `setInputVal` technique, this component doesn't need to have an internal `input` state variable

Reply:
The value attribute of the input tag is assigned by the value in the input variable. (line before onChange = ....)
the internal input state is still necessary.